packages/back-end/src/app.ts

```diff
@@ -365,7 +365,9 @@ app.get("/members", organizationsController.getUsers);
 app.delete("/member/:id", organizationsController.deleteMember);
 app.put("/member/:id/role", organizationsController.putMemberRole);
 app.post("/oauth/google", datasourcesController.postGoogleOauthRedirect);
-app.post("/subscription/start", stripeController.postStartTrial);
+app.post("/subscription/checkout", stripeController.postNewSubscription);
+app.get("/discount-code", stripeController.getDiscountData);
+app.get("/price", stripeController.getPriceData);
```

Comment:
I don't know if we want to expose the discount/price data directly to the front-end.  I feel like the logic to calculate prices and discounts should live in the back-end.  So instead of these, have a single API endpoint that returns exactly what the front-end needs.  The current price per month, the cost to add an additional user, etc.

Reply:
10-4! Just updated the price endpoint to handle these calculations.